tests/scripts/pds_request.py

```diff
@@ -108,7 +108,7 @@ def __init__(
         else:
             self._headers = {
                 'Authorization': f'Bearer {token}',
-                'NHSD-Session-URID': 'ROLE-ID',
+                'NHSD-Session-URID': '555254242102',
```

Comment:
Can we make this more DRY and declare it as a (config?) variable and use that instead of having a magic string repeated everywhere?

Reply:
moved this into config.py file